mentorship ios/Constants/URLStringConstants.swift

```diff
@@ -18,6 +20,16 @@ struct URLStringConstants {
 
     struct MentorshipRelation {
         static let sendRequest: String = baseURL + "mentorship_relation/send_request"
+        static let currentRelation: String = baseURL + "/mentorship_relations/current"
```

Comment:
nit: could take mentorship_relation in common for this struct and add a variable with baseURL + mentorship_relation

Reply:
Do you think this can lead to breakages in the future in the event of different naming of a URL?